preliminary-work/nematic_order.py

```diff
@@ -0,0 +1,40 @@
+import flowermd
+import hoomd
+import gsd
+import numpy as np
+import freud
+
+
+def nematic_order(gsd_file, start=0, stop=None, stride=1):
+      """
+       Calculate nematic order for a coarse-grained system by calculating bond vectors for each bond between two beads.
+       
+       Parameters
+        ----------
+        gsdfile : str
+            Filename for gsd file
+        start : int, default 0
+            Where to start reading the gsd trajectory the system was created
+            with.
+        stop : int, default None
+            Where to stop reading the gsd trajectory the system was created
+            with. If None, will stop at the last frame.
+        stride : int, default 1
+            The step size to use when iterating through start:stop
+        """
+    nm_orders = []
+    directors = [] #adding temporary directors list to export directors generated from nematic function
```

Comment:
I like that we are also returning the director, so let's keep it, rather than it being temporary.